### PR TITLE
Feature: WRITE_OLD_BLOCK_TO_STORAGE hint

### PR DESCRIPTION
**Pull request by odesenfans**
Issue Number: N/A

## Type

- [x] feature
- [ ] bugfix
- [ ] dev (no functional changes, no API changes)
- [ ] fmt (formatting, renaming)
- [ ] build
- [ ] docs
- [ ] testing

## Breaking changes?

- [ ] yes
- [x] no

### CODE REVIEW

**Comment by odesenfans** [comment]
Depends on #109.

File: src/hints/execution.rs [hunk 1]
```diff
@@ -1071,6 +1071,36 @@ pub fn gen_nonce_arg(
...
+    let block_hash_contract_address =
+        get_integer_from_var_name(vars::ids::BLOCK_HASH_CONTRACT_ADDRESS, vm, ids_data, ap_tracking)?.into_owned();
+    let old_block_number =
+        get_integer_from_var_name(vars::ids::OLD_BLOCK_NUMBER, vm, ids_data, ap_tracking)?.into_owned();
+    let old_block_hash = get_integer_from_var_name(vars::ids::OLD_BLOCK_HASH, vm, ids_data, ap_tracking)?.into_owned();
+
+    execution_helper.write_storage_for_address(block_hash_contract_address, old_block_number, old_block_hash).map_err(
+        |_| {
+            HintError::CustomHint(
+                format!("Storage not found for contract {}", block_hash_contract_address).into_boxed_str(),
```
Comment (maciejka) [inline src/hints/execution.rs:1096]: Is 'Storage not found ...' an accurate error message?

Comment (odesenfans) [reply src/hints/execution.rs:1096]: Do you have a better idea? This basically means that the execution helper does not have a storage object for a specific contract. This object should be created when setting up the execution helper, so it's a kind of runtime issue.

Comment (odesenfans) [reply src/hints/execution.rs:1096]: The storage is not written to in this function, the new data is added to an `ongoing_storage_changes` dictionary and this operation will not fail.

**Review by maciejka — APPROVED** [review]

**Pull request merged**